### ISSUE TITLE
Nan's

### ISSUE THREAD

**Issue by ryanpeach** [daynebatten/keras-wtte-rnn#5]
What are your best practices for dealing with NaN's in the output.

I've decided to use masking in the reduce mean for finite values only:

`tf.reduce_mean(tf.boolean_mask(out, tf.is_finite(out))`

I'm thinking of putting in a regularizer or two, and maybe setting max values for a and b?

For instance, if you make your time variable right, the mean of the Weibull should never be greater than 1 I suppose.

Anything else?
